src/components/InpageNav/_inpage-nav.scss

```diff
@@ -53,27 +53,25 @@
 }
 
 .cc-inpage-nav__link {
-  border-left: 2px solid transparent;
-  color: var(--colour-grey-60);
+  @include animated-link($selector: 'span');
```

Comment:
I assume you've done this because mixing the animated link and inpage nav link styles on the same element has side effects?

Reply:
Without span the inpageNav links were looking okay.

![Screenshot 2021-06-07 at 12 03 09](https://user-images.githubusercontent.com/10700103/121006111-70fc6180-c788-11eb-8ae9-9479a40d6e5c.png)

Should I try to add these styles without updating markup?

Reply:
I would be inclined to use whichever way is less confusing.

Reply:
@creido-welly if that's okay with you, I would like to keep it with span for now.